pontoon/administration/management/commands/sync_projects.py

```diff
@@ -0,0 +1,363 @@
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+from bulk_update.helper import bulk_update
+
+from pontoon.administration.files import update_from_repository
+from pontoon.administration.vcs import commit_to_vcs
+from pontoon.base.models import Entity, Locale, Project, Resource, Translation
+from pontoon.base.utils import match_attr
+from pontoon.base.vcs_models import VCSProject
+
+
+class Command(BaseCommand):
+    args = '<project_slug project_slug ...>'
+    help = 'Synchronize database and remote repositories.'
+
+    def log(self, msg, *args, **kwargs):
+        """Log a message to the console."""
+        self.stdout.write(msg.format(*args, **kwargs))
+
+    def info(self, msg, *args, **kwargs):
+        """Log a message to the console if --verbosity=1 or more."""
+        if self.verbosity >= 1:
+            self.log(msg, *args, **kwargs)
+
+    def debug(self, msg, *args, **kwargs):
```

Comment:
These are logging methods probably interesting enough to move into something like a `PontoonBaseCommand` class at some point.